cns/logger/log.go

```diff
@@ -61,8 +61,8 @@ func Close() {
 	}
 }
 
```

Comment:
could you add in a comment why we need to combine both the function?

Reply:
Since, this has passed all the checks, let me avoid updating this PR. I'll add the comment in my next PR. I have made a note of this.